Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -259,3 +280,58 @@ extension BRBLoginViewController: UITextFieldDelegate {
     }
 
 }
+
+extension BRBLoginViewController: UITableViewDataSource {
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        2
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        section == 0 ? 1 : favoriteItems.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if indexPath.section == 0 {
+            let cell = tableView.dequeueReusableCell(withIdentifier: loginId, for: indexPath) as! BRBLoginTableViewCell
+            cell.configure(stackView: stackView)
+            return cell
+        }
+        let cell = tableView.dequeueReusableCell(withIdentifier: favoriteId) as! FavoriteTableViewCell
+        let name = favoriteItems[indexPath.item]
+        cell.configure(
+            name: name,
+            locations: Defaults[\.favoriteFoodLocations][name],
+            favorited: DefaultsKeys.isFavoriteFood(name)
+        )
+        return cell
```

Comment:
Personally I prefer

```
if ... {
    ...
    return cell
} else {
    ...
    return cell
}
```

As you wrote it, it reads like a guard, which suggests that `section == 0` is a rarer case than the `else`. Maybe I'm reading a lot into it, but that's my opinion.